src/components/structures/HomePage.tsx

```diff
@@ -94,7 +96,7 @@ const UserWelcomeTop = () => {
 const HomePage: React.FC<IProps> = ({ justRegistered = false }) => {
     const config = SdkConfig.get();
     const pageUrl = getHomePageUrl(config);
-
+    const homePageWelcomeMsg = SettingsStore.getValue(UIFeature.ShowLiberateToCommunicateText);
```

Comment:
Should the local variable homePageWelcomeMsg be called showLiberateToCommunicateText?

Reply:
This one has been actioned.